akka-stream/src/main/scala/akka/stream/impl/PhasedFusingActorMaterializer.scala

```diff
@@ -368,8 +368,13 @@ private final case class SavedIslandData(islandGlobalOffset: Int, lastVisitedOff
 
   private[this] def createFlowName(): String = flowNames.next()
 
-  /** INTERNAL API */
-  private[akka] val defaultInitialAttributes = {
+  /**
+   * Default attributes for the materializer, are always seen as least specific, so any attribute
+   * specified in the graph or on the stages "wins" over these
+   *
+   * INTERNAL API
+   */
+  private[akka] val defaultAttributes = {
```

Comment:
yes that's definitely good

Reply:
yes, do you mean that this was not the case before?

Reply:
This was the case before, just renamed it to be more clear. I'll port this into the other PR.